src/hash160.rs

```diff
@@ -159,6 +165,25 @@ mod tests {
         assert_tokens(&hash.compact(), &[Token::BorrowedBytes(&HASH_BYTES[..])]);
         assert_tokens(&hash.readable(), &[Token::Str("132072df690933835eb8b6ad0b77e7b6f14acad7")]);
     }
+
+    #[cfg(feature="json")]
+    #[cfg(feature="serde")]
```

Comment:
Can you use `#[cfg(all(feature="json",feature="serde"))]` rather than doubling up `cfg` lines?

Reply:
fixed